packages/web-components/src/index.ts

```diff
@@ -1,3 +1,5 @@
+import '../../core/src/i18n/i18n-setup';
```

Comment:
This is similar to the import in the `react-components` index file, except typescript is able to use the relative import successfully.